composer.json

```diff
@@ -144,6 +145,7 @@
         "pre-pool-create": [
             "scripts\\composer\\DrupalLenientRequirement::changeVersionConstraint"
         ],
+        "pre-autoload-dump": "Google\\Task\\Composer::cleanup",
```

Comment:
This will delete all services except the ones specified in extras.google/apiclient-services so we do not blow out the docker image by 100MB of unused libraries :-)